src/handbook/product/personas/index.md

```diff
@@ -518,4 +518,33 @@ What problem does the persona want to solve, or which benefit does the character
             </ul>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+## Common Use Cases
+
+Based on the Personas, the following section describes the most relevant Use Cases that we identified in exchange with our customers. These Use Cases align with the [Product Strategy](../strategy.md).
+
+1. **Data transformation**
+As a Node-RED User,
```

Comment:
```suggestion
As a Node-RED User,
```

Let's include the persona's, not the `Node-RED` aspect? Not sure, but feels like we should figure out who it's targetting

Reply:
Done